fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/bottomsheet/BottomSheet.kt

```diff
@@ -233,9 +238,11 @@ fun BottomSheet(
     showHandle: Boolean = true,
     slideOver: Boolean = true,
     enableSwipeDismiss: Boolean = false,
+    preventDismissalOnScrimClick: Boolean = false,
     stickyThresholdUpward: Float = 56f,
     stickyThresholdDownward: Float = 56f,
     bottomSheetTokens: BottomSheetTokens? = null,
+    onScrimClick: () -> Unit = {},
```

Comment:
BottomDrawer takes two params, onDismiss and onScrimClick, these two differ on when they run (post closing of drawer or pre), can we do the same thing here, so all scanerios are covered?

Reply:
BottomDrawer doesn't expose onDismiss